src/slice/sort.rs

```diff
@@ -238,11 +243,6 @@ where
             return true;
```

Comment:
If the `SHORTEST_SHIFTING` condition is moved up, it changes this case that would have returned `true`. I haven't fully analyzed that effect, but it's not just about "readability" at that point.

Reply:
You are absolutely right. The behaviour on short sorted slices changes, my bad.

Reply:
OK, then I'll close this - thanks anyway!